catalog/ui/search-ui/search-endpoint/src/main/java/org/codice/ddf/ui/searchui/query/solr/FilteringDynamicSchemaResolver.java

```diff
@@ -35,6 +39,8 @@
 
     public static final String SOURCE_ID = "source-id";
 
+    private static final String EXT_SORT_BY = "EXT_SORT_BY";
```

Comment:
I didn't really have a good logical place to put this definition where it wouldn't break the binary compatibility with the API, or break how we define modules (e.g. don't want to depend on implementation classes outside of a specific module).